packages/material-renderers/src/util/datejs.tsx

```diff
@@ -13,10 +13,31 @@ export const createOnChangeHandler =
   (time: dayjs.Dayjs) => {
     if (!time) {
       handleChange(path, undefined);
-      return;
+    } else if (time.toString() !== 'Invalid Date') {
+      const result = dayjs(time).format(saveFormat);
+      handleChange(path, result);
     }
-    const result = dayjs(time).format(saveFormat);
-    handleChange(path, result);
+  };
+
+export const createOnBlurHandler =
+  (
+    path: string,
+    handleChange: (path: string, value: any) => void,
+    format: string | undefined,
+    saveFormat: string | undefined,
+    rerenderChild: () => void,
+    resetFocus: () => void
```

Comment:
This method does not care why this callback is handed over. The parameter should be called `onBlur` or `callback`